adhocracy4/projects/models.py

```diff
@@ -241,9 +242,11 @@ class Project(ProjectContactDetailMixin,
         ),
         upload_to='projects/tiles',
         blank=True,
-        max_length=300)
+        max_length=512
+    )
```

Comment:
will check this again mB tiles in morning as if it's same test then might break styling

Reply:
It's the name of the image, which is hopefully not displayed anywhere...